src/pencilkit.cs

```diff
@@ -70,6 +86,15 @@ enum PKCanvasViewDrawingPolicy : ulong {
 		PencilOnly,
 	}
 
+	[iOS (17, 0), Mac (14, 0)]
+	[Introduced (PlatformName.MacCatalyst, 17, 0)]
+	[Native]
+	public enum PKContentVersion : long {
+		PKContentVersion1 = 1,
+		PKContentVersion2 = 2,
+		Latest = PKContentVersion2,
```

Comment:
```suggestion
		Version1 = 1,
		Version2 = 2,
		Latest = Version2,
```
How about these names? Sometimes the Swift side of the Apple docs provide alternative names!